packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -5531,6 +5531,7 @@ export interface IModal {
 
 // @public (undocumented)
 export interface IModalProps extends React.ClassAttributes<ModalBase>, IWithResponsiveModeState, IAccessiblePopupProps {
+    allowIosBodyScroll?: boolean;

```

Comment:
Rename this to `allowTouchBodyScroll` or something non-platform specific.

Reply:
this is good point, i will change it to allowTouchBodyScroll, i think i took the name from [_disableIosBodyScroll](https://github.com/OfficeDev/office-ui-fabric-react/blob/2894062b1d0515055c29061dfc6f586f918ca41c/packages/utilities/src/scroll.ts#L86)